mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/impl/MongoDBSchemaDefiner.java

```diff
@@ -156,6 +156,8 @@ private void validateIndexSpecs(SchemaDefinitionContext context) {
 
 					validateIndexSpecsForUniqueColumns( table, indexOptions, forIndexNotReferenced, constraintMethod );
 
+					// keep this order
```

Comment:
What order? The 3 validate methods? Better to add the order at the beginning then instead than in the middle